gotree_test.go

```diff
@@ -0,0 +1,57 @@
+package gotree
+
+import (
+	"testing"
+)
+
+// TestFolderTree should test if reading a folder returns rendered structure as expected
+func TestFolderTree(t *testing.T) {
```

Comment:
I don't like the idea of keeping the testfiles in the project, we should Improve the ReadFolder to be mockable or remove this test for now.